nextflow.config

```diff
@@ -371,7 +371,7 @@ manifest {
     description     = """Assembly, binning and annotation of metagenomes"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '2.4.0dev'
+    version = '2.4.0'
```

Comment:
this is an harshil unworthy indent

Reply:
Don't blame me! Blame tools!!!!

Reply:
Yes, made by tools & requires manual correction. I gave up on that :D